app/models/patient.rb

```diff
@@ -126,51 +118,16 @@ def self.pledged_status_summary(num_days = 7)
     { pledged: outstanding_pledges, sent: sent_total }
   end
 
-  def self.contacted_since(datetime)
-    patients_reached = []
-    all.each do |patient|
-      calls = patient.calls.select { |call| call.status == 'Reached patient' &&
-                                            call.created_at >= datetime }
-      patients_reached << patient if calls.present?
-    end
-
-    # Should we use this or first call?
-    first_contact = patients_reached.select { |patient| patient.initial_call_date >= datetime }
-
-    # hard coding in first_contacts and pledges_sent for now
-    { since: datetime, contacts: patients_reached.length,
-      first_contacts: first_contact.length, pledges_sent: 20 }
-  end
-
-  def recent_calls
-    calls.includes(:created_by).order('created_at DESC').limit(10)
-  end
-
-  def old_calls
-    calls.includes(:created_by).order('created_at DESC').offset(10)
-  end
-
-  def most_recent_note_display_text
-    note_text = most_recent_note.try(:full_text).to_s
-    display_note = note_text[0..30]
-    display_note << '...' if note_text.length > 31
-    display_note
-  end
-
-  def most_recent_note
-    notes.order('created_at DESC').limit(1).first
-  end
-
   # TODO: reimplement once pledge is available
-  #def most_recent_pledge_display_date
-  #  display_date = most_recent_pledge.try(:sent).to_s
-  #  display_date
-  #end
+  def most_recent_pledge_display_date
+   display_date = most_recent_pledge.try(:sent).to_s
+   display_date
+  end
 
   # TODO: reimplement once pledge is available
-  #def most_recent_pledge
-  #  pledges.order('created_at DESC').limit(1).first
-  #end
+  def most_recent_pledge
```

Comment:
whoops. nope. We can remove these since we went a different direction with our pledge. I'm gonna strip 'em.